tools/fetch-sponsors.js

```diff
@@ -227,7 +227,7 @@ async function fetchGitHubSponsors() {
     // return an array in the same format as Open Collective
     const sponsors = organization.sponsorshipsAsMaintainer.nodes
         .map(({ sponsor, tier }) => ({
-            name: sponsor.name,
+            name: sponsor.name || sponsor.login,
```

Comment:
I believe this fixes `null` names in `sponsors.json`, which is fine, but since we already had `null` names in that file (for example, [this one](https://github.com/eslint/eslint.org/blob/415e0bba84b8d205faa5c47924c928b0b0074dba/src/_data/sponsors.json#L1327-L1334)), build is probably failing because of `null` names in `donations.json` so it seems that we should do the same change few lines below:

https://github.com/eslint/eslint.org/blob/415e0bba84b8d205faa5c47924c928b0b0074dba/tools/fetch-sponsors.js#L242